bin/compile

```diff
@@ -48,36 +38,12 @@ if [[ $CLEAR_CACHE -eq 1 ]] ; then
   rm -fr $CACHE_DIR/
 fi
 
-############# Library files ###############################
-
-if [ ! -e $CACHE_DIR/ghc-libs ]; then
```

Comment:
@yingw787 Hi there. Instead of deleting these lines, could you add a conditional(< 6.0.0) for downloading library files so previous versions keep working? I think some users have deployments with the old versions and it would be good to keep backwards compat.

The change related to the binary looks good.